tools/cockpit.spec

```diff
@@ -262,10 +262,12 @@ rm -rf %{buildroot}/debug
 
 %files pcp
 %{_libexecdir}/cockpit-pcp
+/var/lib/pcp/config/pmlogconf/tools/cockpit
 
 %post pcp
 # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1185749
 ( cd /var/lib/pcp/pmns && ./Rebuild -du )
+/usr/share/pcp/lib/pmlogger reload
```

Comment:
What does this do? Does it start pmlogger if not already started? Just double checking.